src/Color.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Chart;
+
+/**
+ * Color generator.
+ *
+ * @todo Replace this with something which would generate infinite colors.
+ * Like https://medium.com/code-nebula/automatically-generate-chart-colors-with-chart-js-d3s-color-scales-f62e282b2b41 or something else
+ */
+class Color
+{
+    /**
+     * We will use these colors in charts.
+     * First can be used as background color and 2nd as border color.
+     *
+     * @const array<int, array<int, string>>
+     */
+    protected const COLORS = [
```

Comment:
This should probably be a property, any reason to be a const?

Reply:
For now it's constant and not changeable, so no use to set it as property, but I guess we can make it changeable/configurable too, so probably setting it as protected property has a point. Will do.